styles.css

```diff
@@ -0,0 +1,19 @@
+body{
+    text-align: center;
+    font-family: Arial;
+    color: blue;
+    max-width: 100%;
+  }
+  h1{
+    font-size: 40px;

```

Comment:
as reviewed, after writing a style there is no skipped line, I would like to kindly suggest that we add a line so that the code looks cleaner